ethstaker_deposit/exceptions.py

```diff
@@ -1,2 +1,9 @@
 class ValidationError(Exception):
     ...
+
+
+class MultiLanguageError(Exception):
```

Comment:
Adding this new exception type will need to be handled in https://github.com/eth-educators/ethstaker-deposit-cli/blob/main/ethstaker_deposit/deposit.py#L128 if it bubbles up.

Reply:
Great catch